src/components/DesktopHeader.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+import withStyles from '@material-ui/core/styles/withStyles';
+import AppBar from '@material-ui/core/AppBar';
+import Button from '@material-ui/core/Button';
+import List from '@material-ui/core/List/List';
+import ListItem from '@material-ui/core/ListItem/ListItem';
+import Menu from '@material-ui/icons/Menu';
+import SwipeableDrawer from '@material-ui/core/SwipeableDrawer/SwipeableDrawer';
+
+import '../App.css';
+
+
+const styles = {
+  moreIcon: {
+    float: 'right',
+    display: 'inline',
+    color: 'white',
+    paddingBottom: 0,
+  },
+
+};
+
+const iOS = process.browser && /iPad|iPhone|iPod/.test(navigator.userAgent);
```

Comment:
Do iPhones or iPods ever get the Desktop view anyways?

Reply:
So this was a problem with Material UI and how iOS devices handle the Swipeable Drawer component. I copied this logic from OSM Quality Web but tracked down a GitHub issue that outlines some of the issues that were happening.

https://github.com/mui-org/material-ui/issues/12057

I'm going to remove this logic for now but when we test on iOS, let's check this out.